packages/instantsearch.js/src/connectors/hierarchical-menu/connectHierarchicalMenu.ts

```diff
@@ -315,12 +315,16 @@ const connectHierarchicalMenu: HierarchicalMenuConnector =
           let items: HierarchicalMenuRenderState['items'] = [];
           let canToggleShowMore = false;
 
+          const getWidgetUiState = this.getWidgetUiState;
```

Comment:
what is the alias used for?

Reply:
Because `_createURL` underneath is not a lambda
I guess I can change it to a lambda though, doesn't seem like there would be repercussions